### DISCUSSION TITLE
‘Act of Directive Communication’ definition mistakenly requires hearer

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [CommonCoreOntology/CommonCoreOntologies#369] [Announcements]
One can directively communicate (e.g., advise or order something, to take just a few of the examples given in the rdfs:comment) in a non-auditory manner, so ‘hearer’ seems mistaken. Better to use ‘audience’, I suspect.

**Comment by gregfowlerphd** [comment]
I think there are other issues with this class--in particular, with the relation between it and its subclasses--that might necessitate other revisions to the definition. So revising now would be premature.

(@cameronmore: Tagging you here since you gave the original post a thumbs up, suggesting you agree about the problem stated and perhaps about its solution.)

**Comment by swartik** [comment]
Recognizing what @gregfowlerphd just said about delaying revision, I still want to mention that the Oxford Language Dictionary defines "audience" as follows:

> The assembled spectators or listeners at a public event, such as a play, movie, concert, or meeting.

That interpretation of "audience" isn't much better than "hearer". The following definition for Act of Directive Communication, although a little obscure, possibly is closer to the intended meaning:

> An Act of Communication that is intended to cause one or more Agents to receive a Directive Information Content Entity prescribing some Planned Act.

"Prescribing some Planned Act" covers "take a particular action" in the current definition. However, from the examples in the rdfs:comment, I'm not clear whether an Act of Directive Communication always intends to take an action. For example, is "instructing" a synonym for "ordering", or is it what a teacher does to students?

**Comment by gregfowlerphd** [comment]
I'm not sure what to think @swartik's point concerning 'audience'. I agree that the sense in which I was using it is a semi-technical one that doesn't match the ordinary sense captured by the OLD definition he quotes. However, 'audience' is already used in that semi-technical sense elsewhere in the Event Ontology. See the definitions for Act of Personal Communication, Act of Deceptive Communication, and Act of Inviting. (Moreover, its other uses in the Event Ontology also don't seem to conform to the OLD definition.)

In my previous comment, I noted that there are other issues with the Act of Directive Communication class that might necessitate further revisions to its definition:

> An Act of Communication that is intended to cause the hearer to take a particular action.

I'll discuss these below.

First, not all Acts of Advising--which, by definition, are 'performed by providing advice or counsel to another agent'--seem to be intended to cause the "hearer" to take a _particular_ action. A student's faculty advisor might lay out for the student _multiple_ paths for the student to choose between based on their interests, availability, etc.

Second, Acts of Requesting (which are 'performed by asking for something') are subject to similar concerns. Suppose I ask you to accompany me to the movies or to coffee. In that case, there's no _particular_ action I intend to cause you to take, but it would seem I've still performed an Act of Requesting.

Third, a related problem arises for Acts of Warning (which are 'performed by indicating a possible or impending danger, problem, or other unpleasant situation'). Indeed, in the typical case, an act of warning is not be intended to cause the "hearer" to take a particular action. Instead, it's intended to cause them _not_ to take a particular action.

Thus, given the current definition for Act of Directive Communication, it would appear that Act of Advising, Act of Requesting, and Act of Warning aren't subclasses of ADC, contra the asserted subclass relations. (Is the same true of the fourth asserted subclass, Act of Commanding? Perhaps, we accept the possibility of disjunctive commands.)

(@cameronmore: Tagging you here for the same reason as before.)

**Comment by swartik** [comment]
Well, this issue of what "audience" means points to a larger problem. If an ontology exists to give standard meaning to a set of concepts, and those concepts are supplemented by natural-language explanations, then do we also agree there should be standards on how to write the explanations? CCO already has some syntactic standards (e.g., "A _class_ that _elaboration_"). Should CCO also state a definitive source to use for the meanings of words in explanations?

I don't want to fight battles over whether "audience" is acceptable. When I think of "audience", the first thing I think of is people in an auditorium, receiving audio. You can retort with any number of counterexamples, from silent movies to John Cage's _4'33" for piano_. And those aren't even figurative interpretations. I'm only hoping we can come up with a way to adjudicate whether a word is acceptable, along with helping CCO users interpret definitions.

**Comment by neilotte** [comment]
@gregfowlerphd  The issues raised in this thread require a more comprehensive review. For this reason, I am converting this into a discussion where we can work to identify a project plan to implement decisions made here. Thanks!

**Comment by PhiBabs935** [comment]
@neilotte @gregfowlerphd 

Why not just use 'recipient' (or 'recipient Agent') in the skos:definition for 'Act of Directive Communication?

'recipient' after all is the language used in the object property 'receives', which has range 'Act of Communication':

"x receives y iff x is an instance of Agent and y is an instance of Act Of Communication, such that x is the recipient and decoder of the InformationContentEntity intended for communication in y."

If one was modeling an instance of Act of Directive Communication toward a given Agent one would assert: 
           ActOfDirectiveCommunication1 _has recipient_ Agent1 
or:      Agent1 _receives_ ActOfDirectiveCommunication1

which implies that Agent1 is the recipient and decoder of the ICE intended for communication in ActOfDirectiveCommunication1
